netci.groovy

```diff
@@ -3097,29 +3072,18 @@ def static CreateWindowsArmTestJob(def dslFactory, def project, def architecture
                 // as save the current errorlevel on the same line. This works because CMD evaluates the %errorlevel%
                 // variable expansion (or any variable expansion on the line) BEFORE it executes the ENDLOCAL command.
                 // Note that the ENDLOCAL also undoes the pushd command, but we add the popd here for clarity.
-                addCommand("popd & ENDLOCAL & set __save_smarty_errorlevel=%errorlevel%")
-
-                // ZIP up the smarty output, no matter what the smarty result.
-                addCommand("powershell -NoProfile -Command \"Add-Type -Assembly 'System.IO.Compression.FileSystem'; [System.IO.Compression.ZipFile]::CreateFromDirectory('.\\bin\\tests\\${osGroup}.${architecture}.${configuration}\\Smarty.run.0', '.\\bin\\tests\\${osGroup}.${architecture}.${configuration}\\Smarty.run.0.zip')\"")
-
-                addCommand("echo %errorlevel%")
-                addCommand("dir .\\bin\\tests\\${osGroup}.${architecture}.${configuration}")
+                addCommand("popd & ENDLOCAL & set __save_runtest_errorlevel=%errorlevel%")
```

Comment:
There's no need for this anymore (or the giant comment above) since there are no commands between the run and the `exit`. Just do `exit /b %errorlevel%`

Reply:
Addressed